src/server/rest/v1/service/AssetService.ts

```diff
@@ -33,27 +31,19 @@ export default class AssetService {
   public static async handleGetAssetConsumption(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
     // Check if component is active
     UtilsService.assertComponentIsActiveFromToken(req.user, TenantComponents.ASSET,
-      Action.LIST, Entity.ASSET, MODULE_NAME, 'handleGetAssetConsumption');
+      Action.READ_CONSUMPTION, Entity.ASSET, MODULE_NAME, 'handleGetAssetConsumption');
     // Filter
     const filteredRequest = AssetValidator.getInstance().validateAssetGetConsumptionsReq(req.query);
     UtilsService.assertIdIsProvided(action, filteredRequest.AssetID, MODULE_NAME,
```

Comment:
I created a new action for READ_CONSUMPTION so that it had it's own authorizations and was only applicable for dynamic assets:

```   
const asset = await AssetService.getAssetFromStorage(req.tenant, filteredRequest.AssetID, { dynamicOnly: true }, actionOnAssetAuthorizationFilter);
```